packages/bot-web-ui/src/components/load-modal/__tests__/load-modal.spec.tsx

```diff
@@ -82,12 +86,63 @@ describe('LoadModal', () => {
         expect(screen.getByRole('button', { name: 'Open' })).toBeInTheDocument();
     });
 
+    it('should render load modal preview on file upload and on click of close should close the preview', () => {
+        mock_store.ui.is_mobile = false;
+        render(<LoadModal />, { wrapper });
+        mock_DBot_store?.load_modal.setActiveTabIndex(1);
+
+        mock_DBot_store?.load_modal.setLoadedLocalFile(new File([''], 'test-name', { type: 'text/xml' }));
+
+        const close_button = screen.getByTestId('dt_load-strategy__local-preview-close');
+        expect(close_button).toBeInTheDocument();
+
+        fireEvent.click(close_button);
```

Comment:
@rupato-deriv can we use userEvent here?

Reply:
done ✅